hindsight_server/views/timeline_view.py

```diff
@@ -21,114 +20,158 @@
 
 from db import HindsightDB
 
-local_timezone = tzlocal.get_localzone()
-video_timezone = ZoneInfo("UTC")
-
 @dataclass
 class Screenshot:
     image: np.array
     text_df: pd.DataFrame
     timestamp: datetime.timestamp
 
+local_timezone = tzlocal.get_localzone()
+video_timezone = ZoneInfo("UTC")
+timeline_scrollbar_height = 50
+
+def get_images_df(db: HindsightDB, front_camera):
+    """Gets a DataFrame of all images at the time of inititation"""
```

Comment:
Yeah good call pulling these out. A lot of them could be moved to some more specific utils. Using rects overlap a lot for the parsing but not worried about that for now